apps/core/src/components/Validator.tsx

```diff
@@ -14,17 +16,21 @@ import {
 import { formatAddress } from '@iota/iota-sdk/utils';
 
 interface ValidatorProps {
-    isSelected: boolean;
+    isSelected?: boolean;
     address: string;
+    type?: CardType;
+    showApy?: boolean;
     showActiveStatus?: boolean;
-    onClick?: (address: string) => void;
+    onClick?(): void;
```

Comment:
I wonder how did this even compile
```suggestion
    onClick?: () => void;
```